.gitignore

```diff
@@ -70,3 +70,12 @@
 !LICENSE
 !README.md
 !VERSION
+
+# Ignore Lock & dirLock files
+covalent_ui/dask-worker-space/*.*lock
+covalent_ui/app/dask-worker-space/*.*lock
+
+# Ignore .sonarwork files
```

Comment:
@wjcunningham7 We are running Sonar cube locally for continuous inspection of code quality. So there are some files generated by Sonar cube and to ignore those files, we have added it to git ignore.